zengo/migrations/0005_ticket_priority.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.10 on 2020-02-28 18:25
+
+from django.db import migrations
+import konst.models.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zengo", "0004_zendeskuser_alias"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="ticket",
+            name="priority",
+            field=konst.models.fields.ConstantChoiceCharField(
+                choices=[
+                    ("urgent", "urgent"),
+                    ("high", "high"),
+                    ("normal", "normal"),
+                    ("low", "low"),
+                ],
+                default="low",
```

Comment:
It's reasonable to choose a default rather than have this field as nullable, given all tickets have a priority, however I think "normal" might be more appropriate.